nvflare/private/fed/app/client/client_train.py

```diff
@@ -75,6 +88,11 @@ def main(args):
         )
         conf.configure()
 
+        decomposer_module = ConfigService.get_str_var(
```

Comment:
We shouldn't load custom decomposers since this is CP.

Reply:
We always do this fobs initialize.

https://github.com/yhwen/NVFlare/blob/fobs_auto_register/nvflare/private/fed/app/client/client_train.py#L82

Reply:
We should rethink about this. Custom decomposers shouldn't be loaded for SP and CP.

Reply:
Removed the custom decomposer register for CP.